app/layout.tsx

```diff
@@ -22,7 +22,7 @@ export default function RootLayout({
         className={inter.className + " bg-black flex flex-col min-h-screen"}
       >
         <Navbar />
-        <div className="flex-1">{children}</div>
+        <div className="flex-1 " style={{width: "95%", margin: "0 auto", textAlign: "center",} } >{children}</div>
```

Comment:
This limits full width images to 95% of screen as well. Do we want that?

Reply:
Hmm true, I think for now I'll leave it be, however post deployment we can potentially figure out how to do this efficiently for the text sections